Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -113,5 +116,21 @@ public IEnumerable<LayerAsset> GetAssets()
         {
             return rootLayer.GetAssets();
         }
+
+        public void AddFunctionality(FunctionalityData data)
+        {
+            if (!functionalities.Contains(data))
```

Comment:
Moet dit niet een check op de Id zijn ipv of het object al gebruikt wordt? Als je een project zou inladen dan krijg je andere object hashes en werkt deze vergelijking toch niet?